_posts/2016-03-12-make-your-data-obey.markdown

```diff
@@ -0,0 +1,200 @@
+---
+layout: post
+title: "Make Your Data Obey"
+date: 2016-03-12 10:00:00
```

Comment:
Reminder to update with publish. The Octopress 3 CLI can generate with TZ offset, from _dtrafs, if desired using the publish command.

CLI ref: https://github.com/octopress/octopress/blob/master/README.md